### ISSUE TITLE
Let us select status type for Discord RPC

### ISSUE THREAD

**Issue by kyoyacchi** [Quickdesh/Animiru#17]
### Describe your suggested feature

Hi. Since you've added discord rpc feature, it would be good to have a feature that lets us to select status type, because it shows online for others even though i'm dnd on discord. 
![Screenshot_20220815-122305](https://user-images.githubusercontent.com/63583961/184610687-78f36a69-caec-46bd-8cf1-5938cf6401fa.jpg)
(This ss is from my alt account)

### Other details

Now that's interesting. It's continues to show im watching anime even tho im not watching.

---
I mean
![Screenshot_20220815-122951_1_1](https://user-images.githubusercontent.com/63583961/184611849-f38cb797-4aad-4f94-b3ab-a810f8aefe12.jpg)
![Screenshot_20220815-122954_1](https://user-images.githubusercontent.com/63583961/184611860-6fa6f7f3-19b7-40ce-b588-d7f35fbffec0.jpg)

Like this

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/jmir1/aniyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.13.4.1](https://github.com/Quickdesh/Animiru/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Quickdesh** [comment]
Ye ye, is planned. I'll add the option in the next update!

Also about it showing that you are online, The discord api takes a little while to update. If app is running in the background, Its set to run for an entire minute before closing the RPC.

TL;DR, 
1. Next update I'll add it 
2. Is intended behaviour

**Comment by Quickdesh** [comment]
Oh um also


BLOODY 140 PINGS!!

**Comment by Quickdesh** [comment]
added in `7dd23a9`
